### PR TITLE
Spanish Translation Created

### PR DESCRIPTION
**Pull request by ACKino**

### CODE REVIEW

File: translations/es.ts [hunk 1]
```diff
@@ -0,0 +1,1639 @@
...
+    <message>
+        <location filename="../src/widget/form/addfriendform.cpp" line="38"/>
+        <source>Message</source>
+        <comment>The message you send in friend requests</comment>
+        <translation>Mensaje</translation>
+    </message>
+    <message>
+        <location filename="../src/widget/form/addfriendform.cpp" line="39"/>
+        <source>Send friend request</source>
+        <translation>Mandar solicitud de Amig@</translation>
```
Comment (dubslow) [inline translations/es.ts:101]: Dat typ@ though

Edit: oh snap is this a common thing on the internet?

File: translations/i18n.pri [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # For autocompiling qm-files.
 
-TRANSLATIONS = translations/bg.ts \
+TRANSLATIONS = translations/es.ts \
```
Comment (zetok) [inline translations/i18n.pri:3]: Should be in alphabetical order

Comment (dubslow) [inline translations/i18n.pri:3]: The build file order doesn't matter, but it does matter in the UI list.

Comment (zetok) [inline translations/i18n.pri:3]: @dubslow I beg to differ. Keeping things organized helps maintain whole thing in good shape, beside making it easier to navigate and look through. Once it starts getting messy…

File: translations/es.ts (hunk 1 above)
Comment (ACKino) [inline translations/es.ts:101]: Yes. It avoids being gender specific. I figured tox users might appreciate that.
Alternatives would be to write amigo/amiga or avoid the word friend altogether and use contact which only translates as contacto, contacta being the verb.

File: translations/i18n.pri (hunk 2 above)
Comment (ACKino) [inline translations/i18n.pri:3]: will fix build file and UI list on next translation update, if nobody else has by then,

**Pull request merged**